gemini/js-web-genkit/package.json

```diff
@@ -0,0 +1,25 @@
+{
+    "name": "genkit-gemin-web-example",
+    "version": "1.0.0",
+    "description": "This is a simple demonstration web app using the [Firebase Genkit SDK](https://github.com/firebase/genkit) to generate recipes for images.",
+    "main": "index.js",
+    "scripts": {
+        "dev": "GENKIT_ENV=dev npx tsx --watch server.ts"
+    },
+    "dependencies": {
+        "@genkit-ai/googleai": "^0.9.0-rc || ^0.9",
+        "genkit": " ^0.9.0-rc || ^0.9",
+        "base64-js": "^1.5.1",
```

Comment:
This can be removed

Reply:
which one can be removed?

Reply:
Both base 64 and markdown it! It uses the can for markdown and base 64 via node api

Reply:
Ah got it, updated!